packages/cli/src/Db.ts

```diff
@@ -6,7 +6,7 @@ import { DataSource as Connection } from '@n8n/typeorm';
 import { ErrorReporterProxy as ErrorReporter } from 'n8n-workflow';
 
 import { inTest } from '@/constants';
-import { wrapMigration } from '@db/utils/migrationHelpers';
+import { wrapMigration } from '@/databases/utils/migration-helpers';
```

Comment:
`@db` → `@/databases` comes automatically from symbol rename via Intellisense. We can change it back to `@db` if need be.

Reply:
Could we have just one alias and stick to that

Reply:
I think this will need a lint rule to enforce. Right now, Intellisense sometimes autocompletes imports with relative imports, sometimes with `@/databases` but apparently never with the alias.

Reply:
I started with `@db` because I think I wanted to extract all DB stuff out into a separate package. I don't know if any of the reasons I had back then make sense anymore. 
perhaps we should just drop `@db` and replace it all with `@/databases`.